source/docs/software/advanced-controls/introduction/tuning-bang-bang-controller.rst

```diff
@@ -0,0 +1,52 @@
+Tuning a Bang-Bang Controller
+=============================
+
+A "Bang-Bang" controller is a simple controller which applies a binary (present/not-present) force to a mechanism to try to get it closer to a setpoint. 
+
+:ref:`WPILib provides an implementation <docs/software/advanced-controls/controllers/bang-bang:Bang-Bang Control with BangBangController>` for bang-bang control as well. This page includes a more in-depth description.
+
+
+General Tuning Techniques
+-------------------------
+
+Bang-Bang controllers do not have any parameters to tune. They either push the mechanism toward the setpoint from the "too-low" side, or allow it to drift back toward the setpoint from the "too-high" side.
```

Comment:
yeah, we can distinguish between asymmetric bang-bang and symmetric bang-bang here.  Asymmetric is way more usable because systems that want meaningful control effort in both directions generally do not want an ugly response discontinuity at 0.

Reply:
For the sake of introduction.... maybe just ignore the difference and only present the Asymetric version.... and ignore the fact that the command voltage can be tuned?

A lot of this page is just to show "look, you can get kinda close by barely thinking at all, but it might not work because of XYZ. If it doesn't work, keep going".

Reply:
That's fair.  Just word it carefully so that the article doesn't lie (except by omission, and with links where things are omitted).